azure_pipelines/automation.yml

```diff
@@ -98,4 +98,50 @@ jobs:
       targetPath: '$(Agent.BuildDirectory)/s/test_output/'
       artifact: 'TestOutputs'
       publishLocation: 'pipeline'
+  - task: UsePythonVersion@0
+    displayName: Use Python 3.x
+  - bash: python3 -m pip install github3.py==2.0.0
+    displayName: Install the github3.py REST API client for Python
+  - task: PythonScript@0
+    displayName: Create Issue if Pipeline fails
+    condition: failed()
+    inputs:
+      scriptSource: inline
+      script: |
+        import github3
+        import requests
+        import textwrap
+        from pprint import pprint
+        from subprocess import run, PIPE
+        import random
+
+        auth_header = ""
+        try:
+            auth_header = run(["git", "config", "http.$(Build.Repository.URI).extraheader"], text=True, check=True, stdout=PIPE).stdout.strip()
+        except Exception as e:
+          pprint(e)
+        class HTTPHardcodedHeaderAuth(requests.auth.AuthBase):
+            def __call__(self, r):
+                r.headers['Authorization'] = auth_header.replace("AUTHORIZATION: ", "")
+                return r
+
+        github = github3.GitHub()
+        github.session.auth = HTTPHardcodedHeaderAuth()
+        github_org, repo = "$(Build.Repository.Name)".split("/")
+        
+        # Get the members of AppleIdentityTeam in AzureAD organization 
+        apple_id_team = github.organization('AzureAd').team(3752064)
+        members = list(apple_id_team.members(role="all"))
+        
+        # Pick random member from team to assign issue to
```

Comment:
:)

Reply:
Github allows assigning a team as reviewers on PRs but doesn't allow assigning issue to a team :(

Reply:
I guess it doesn't matter to whom the issue is assigned to, OCE will assign it to themselves during triage anyway

Reply:
Agreed